Runner/suites/Kernel/FunctionalArea/baseport/USBHost/run.sh

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+#Setup requires at least one USB peripheral connected to USB port that supports Host mode function
+
+#!/bin/sh
+# Import test suite definitions
+$(pwd)/init_env
+TESTNAME="USBHost"
+
+#import test functions library
+. "$(dirname "$0")/../../utils/functestlib.sh"
```

Comment:
In other scripts this is imported as 
```
. "${TOOLS}"/functestlib.sh
```
Maybe we should keep the same convention here.